.rubocop.yml

```diff
@@ -30,3 +30,7 @@ Style/IndentationConsistency:
 
 Style/RedundantReturn:
   Enabled: true
+
+Style/StringLiterals:
+  Enabled: true
+  EnforcedStyle: single_quotes
```

Comment:
This is the default, idea also warns about `"`, personally I am used to write `'` when I don't need `"`. It's super easy to fix `rubocop --only Style/StringLiterals --auto-correct` (or just `-a`). It's probably easier to parse and translate, if it has a measurable effect than we should definitely keep `'`. (I would have excluded it if there wasn't the autocorrect feature.)